resources/carrier/en/380.txt

```diff
@@ -13,19 +13,26 @@
 # limitations under the License.
 
 # Data taken from:
-# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DB0004PDFE.pdf
+# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DB0005PDFE.pdf
```

Comment:
Can you change the URL as below. This is dynamic and we can't keep changing it always.

\# Data taken from: https://www.itu.int/oth/T02020000DB/en

Reply:
Fixed.